### PR TITLE
feat: add compatibility for 17.41.37

### PR DESCRIPTION
**Pull request by OxrxL**

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/youtube/layout/pivotbar/shortsbutton/patch/ShortsButtonRemoverPatch.kt [hunk 1]
```diff
@@ -63,7 +64,7 @@ class ShortsButtonRemoverPatch : BytecodePatch(
         val enumScanResult = fingerprintResults[0]
         val buttonViewResult = fingerprintResults[1]
 
-        val enumHookInsertIndex = enumScanResult.startIndex
+        val enumHookInsertIndex = enumScanResult.startIndex + 2
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/layout/pivotbar/shortsbutton/patch/ShortsButtonRemoverPatch.kt:67]: Will this not cause issues with existing versions of YouTube?

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: src/main/kotlin/app/revanced/patches/youtube/layout/pivotbar/shortsbutton/patch/ShortsButtonRemoverPatch.kt (hunk 1 above)
Comment (OxrxL) [reply src/main/kotlin/app/revanced/patches/youtube/layout/pivotbar/shortsbutton/patch/ShortsButtonRemoverPatch.kt:67]: Nope...this is based on a fingerprint equal on 17.36.37 and 17.41.37.

Comment (oSumAtrIX) [reply src/main/kotlin/app/revanced/patches/youtube/layout/pivotbar/shortsbutton/patch/ShortsButtonRemoverPatch.kt:67]: Please add the changes to the fingerprint.

**Comment by E85Addict** [comment]
Heads up this error occurs.
SEVERE: video-ads failed:
app.revanced.patcher.patch.PatchResultError
        at app.revanced.patcher.Patcher.executePatches$executePatch(Patcher.kt:346)
        at app.revanced.patcher.Patcher.access$executePatches$executePatch(Patcher.kt:37)
        at app.revanced.patcher.Patcher$executePatches$1.invokeSuspend(Patcher.kt:362)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlin.sequences.SequenceBuilderIterator.hasNext(SequenceBuilder.kt:127)
        at app.revanced.utils.patcher.PatcherKt.applyPatchesVerbose(Patcher.kt:92)
        at app.revanced.cli.patcher.Patcher.start$revanced_cli(Patcher.kt:25)
        at app.revanced.cli.command.MainCommand.run(MainCommand.kt:160)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1939)
        at picocli.CommandLine.access$1300(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at app.revanced.cli.main.MainKt.main(Main.kt:7)
Caused by: java.lang.NullPointerException
        at app.revanced.patches.youtube.ad.video.patch.VideoAdsPatch.execute(VideoAdsPatch.kt:44)
        at app.revanced.patches.youtube.ad.video.patch.VideoAdsPatch.execute(VideoAdsPatch.kt:22)
        at app.revanced.patcher.Patcher.executePatches$executePatch(Patcher.kt:342)
        ... 16 more

**Comment by OxrxL** [comment]
> Heads up this error occurs.
> 
> SEVERE: video-ads failed: app.revanced.patcher.patch.PatchResultError at app.revanced.patcher.Patcher.executePatches$executePatch(Patcher.kt:346) at app.revanced.patcher.Patcher.access$executePatches$executePatch(Patcher.kt:37) at app.revanced.patcher.Patcher$executePatches$1.invokeSuspend(Patcher.kt:362) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlin.sequences.SequenceBuilderIterator.hasNext(SequenceBuilder.kt:127) at app.revanced.utils.patcher.PatcherKt.applyPatchesVerbose(Patcher.kt:92) at app.revanced.cli.patcher.Patcher.start$revanced_cli(Patcher.kt:25) at app.revanced.cli.command.MainCommand.run(MainCommand.kt:160) at picocli.CommandLine.executeUserObject(CommandLine.java:1939) at picocli.CommandLine.access$1300(CommandLine.java:145) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358) at picocli.CommandLine$RunLast.handle(CommandLine.java:2352) at picocli.CommandLine$RunLast.handle(CommandLine.java:2314) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179) at picocli.CommandLine$RunLast.execute(CommandLine.java:2316) at picocli.CommandLine.execute(CommandLine.java:2078) at app.revanced.cli.main.MainKt.main(Main.kt:7) Caused by: java.lang.NullPointerException at app.revanced.patches.youtube.ad.video.patch.VideoAdsPatch.execute(VideoAdsPatch.kt:44) at app.revanced.patches.youtube.ad.video.patch.VideoAdsPatch.execute(VideoAdsPatch.kt:22) at app.revanced.patcher.Patcher.executePatches$executePatch(Patcher.kt:342) ... 16 more

I know...oSum working on a new patch, to fix video ads without generate problems on buffering.

**Pull request merged**